src/localStorage.js

```diff
@@ -3,14 +3,35 @@ export const saveLogin = state => {
     const serializedstate = JSON.stringify({ authentication: state });
     localStorage.setItem("authentication", serializedstate);
   } catch (err) {
-    //Nothing should happen.
+    return err;
+  }
+};
+
+export const saveTeams = state => {
+  try {
+    const serializedstate = JSON.stringify({ teams: state });
+    localStorage.setItem("teams", serializedstate);
+  } catch (err) {
+    return err;
```

Comment:
Still not clear what you are gonna do when saving fails. Do you return feedback to the user?

Reply:
At the moment there is no feedback shown to the user. This was an implementation that was already developed.